### DISCUSSION TITLE
Problems with detecting an Internet connection when installing Windows on macOS

### DISCUSSION THREAD

**Discussion by DascalD** [utmapp/UTM#5070] [UTM Help (macOS)] [answered]
**Describe the issue**  
I tried to create a virtual machine, everything went well and there were no problems, I already created it, but after the moment when you need to select a country, language, and keyboard layout, there is a stage at which it says that there is no Internet connection, although it is and it works, I checked in the browser, I tried to connect to another network, but the result is the same, earlier at this stage there was a checkmark to continue without connecting to the Internet, but now it is gone, and I went to the site that was offered to solve the problem, but did not find anything that could help, because of this problem i can't go further

**Configuration**  
* UTM Version: Version 4.1.5
* macOS Version: macos ventura
* Mac Chip (Intel, M1, ...): M1

**Comment by ideologysec** [comment]
https://github.com/utmapp/UTM/issues/4818

**Comment by osy** [comment, marked as answer]
https://docs.getutm.app/guides/windows/#networking-does-not-work
